third_party/closure-compiler/externs/web_animations.js

```diff
@@ -70,6 +70,53 @@ class Animation {
   reverse() {}
 }
 
+/**
+ * See https://developer.mozilla.org/en-US/docs/Web/API/Animation
+ * @interface
+ * @extend {EventTarget}
```

Comment:
extends `EventTarget`? doesn't seem right

Reply:
Removed in latest patch.